bigtable/client/internal/instance_admin.cc

```diff
@@ -57,6 +57,24 @@ std::vector<btproto::Instance> InstanceAdmin::ListInstances(
   return result;
 }
 
+btproto::Instance InstanceAdmin::GetInstance(std::string const& instance_id,
+                                             grpc::Status& status) {
+  // Copy the policies in effect for the operation.
+  auto rpc_policy = rpc_retry_policy_->clone();
+  auto backoff_policy = rpc_backoff_policy_->clone();
+
+  // Build the RPC request, try to minimize copying.
+  btproto::GetInstanceRequest request;
+  // Set Instance ID
```

Comment:
Remote comment or change to what it actually does (setting the instance **name**).